File: web3rpc/rpc-specs/code-samples/python/personal/replaceRawKey.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

keyData = "24c34f686a5848edb19180fb723b5db21c626f253e8b63bf8a0054ea67852c0a"
oldPassphrase = "helloWorld"
newPassphrase = "helloWorld"

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.replace_raw_key(keyData, oldPassphrase, newPassphrase)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/replaceRawKey.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const keyData = "24c34f686a5848edb19180fb723b5db21c626f253e8b63bf8a0054ea67852c0a";
    const oldPassphrase = "hello@123";
    const newPassphrase = "hello@123";

    sdk.personal.replaceRawKey(keyData, oldPassphrase, newPassphrase, {}, (err, data, response) => {
        console.log(data);
    });
}
)()